roboteam_ai/src/world/Ball.h

```diff
@@ -15,11 +15,13 @@ namespace world {
 
 class WorldData;
 class Robot;
-class Ball {
+class Ball : public std::enable_shared_from_this<Ball> {
```

Comment:
brr I really dont like the enable_shared_from_this in general. Avoid this at all costs.